templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -225,25 +223,14 @@
       % endfor
       EOF
 
-      # The symbol prefixing mechanism is performed by redefining BoringSSL symbols with "#define
-      # SOME_BORINGSSL_SYMBOL GRPC_SHADOW_SOME_BORINGSSL_SYMBOL". Unfortunately, some symbols are
-      # already redefined as macros in BoringSSL headers in the form "#define SOME_BORINGSSL_SYMBOL
-      # SOME_BORINGSSL_SYMBOL" Such type of redefinition will cause "SOME_BORINGSSL_SYMBOL redefined"
-      # error when using together with our prefix header. So the workaround in the below lines removes
-      # all such type of #define directives.
-      sed -i'.back' '/^#define \\([A-Za-z0-9_]*\\) \\1/d' include/openssl/*.h
-      # Remove lines of the format below for the same reason above
-      #     #define SOME_BORINGSSL_SYMBOL ${"\\"}
-      #         SOME_BORINGSSL_SYMBOL
-      sed -i'.back' '/^#define.*\\\\$/{N;/^#define \\([A-Za-z0-9_]*\\) *\\\\\\n *\\1/d;}' include/openssl/*.h
+      # Grab prefix header from Github repo
+      curl -o include/openssl/boringssl_prefix_symbols.h -L https://raw.githubusercontent.com/grpc/grpc/master/src/objective-c/boringssl_prefix_headers/boringssl_prefix_symbols-${boringssl_commit}.h
```

Comment:
I understand the challenge here, but this downloads the right version of boringssl_prefix_symbols (depending on boringssl's SHA) from master of our repository. That means we'd have to keep an ever-growing list of boringssl shadow headers in master of our repository forever (one version for each SHA we use), and that's not a good solution.
 Also I don't like that the download command can fail and that can lead to random errors when building.
Would inlining a base64  gzip compressed version of boringssl_prefix_symbols.h fit in the podspec? I know it's dirty but it's better than the challenges of this approach.
Checking in a generated version of boringssl_prefix_symbols.h into our repo is fine, but then regenerating project could inline the base64 compressed version of it into the podspec (which make the podspec standalone).

Reply:
I tried this and 
`gzip -c boringssl_prefix_symbols.h | base64 >boringssl_prefix_symbols.h.gz.base64` gives you a file 37KB in size.
That should be good enough for us.

```
$ ls -lh
total 320K
-rw-r----- 1 jtattermusch primarygroup 278K Nov 21 08:34 boringssl_prefix_symbols-7f02881e96e51f1873afcf384d02f782b48967ca.h
-rw-r----- 1 jtattermusch primarygroup  37K Nov 21 08:36 boringssl_prefix_symbols.h.gz.base64
```

Reply:
I'll give it a try